web/src/components/ToolbarItem.tsx

```diff
@@ -39,14 +46,32 @@ type ToolbarItemProps = {
   icon: string
   text: string
   id?: string
+  isDisabled?: boolean
 } & ItemProps
 
-const ToolbarItem = ({ href, text, icon, onClick, id }: ToolbarItemProps): ReactElement => (
-  // @ts-expect-error wrong types from polymorphic 'as', see https://github.com/styled-components/styled-components/issues/4112
-  <StyledToolbarItem as={onClick ? Button : undefined} id={id} href={href} onClick={onClick} label={text}>
-    <StyledIcon src={icon} />
-    <StyledSmallViewTip>{text}</StyledSmallViewTip>
-  </StyledToolbarItem>
-)
+const ToolbarItem = ({ href, text, icon, isDisabled = false, onClick, id }: ToolbarItemProps): ReactElement => {
+  const { t } = useTranslation('categories')
+  const toolTipId = spacesToDashes(text)
+  const styledToolbarItem = (
+    <StyledToolbarItem
+      as={onClick ? Button : undefined}
+      id={id}
+      // @ts-expect-error wrong types from polymorphic 'as', see https://github.com/styled-components/styled-components/issues/4112
+      href={isDisabled ? null : href}
+      onClick={isDisabled ? null : onClick}
+      label={text}
+      disabled={isDisabled}>
+      <StyledIcon src={icon} disabled={isDisabled} />
+      <StyledSmallViewTip>{text}</StyledSmallViewTip>
+    </StyledToolbarItem>
+  )
+  return isDisabled ? (
+    <Tooltip id={toolTipId} tooltipContent={t('disabledPdf')}>
+      {styledToolbarItem}
+    </Tooltip>
+  ) : (
+    <>{styledToolbarItem}</>
+  )
```

Comment:
This way you can render a `div` instead of an `a` tag:
```suggestion
  if (isDisabled) {
    return (
      <Tooltip id={toolTipId} tooltipContent={t('disabledPdf')}>
        {/* @ts-expect-error wrong types from polymorphic 'as', see https://github.com/styled-components/styled-components/issues/4112 */}
        <StyledToolbarItem as='div' id={id} label={text} disabled>
          <StyledIcon src={icon} disabled />
          <StyledSmallViewTip>{text}</StyledSmallViewTip>
        </StyledToolbarItem>
      </Tooltip>
    )
  }

  return (
    <StyledToolbarItem
      as={onClick ? Button : undefined}
      id={id}
      // @ts-expect-error wrong types from polymorphic 'as', see https://github.com/styled-components/styled-components/issues/4112
      href={href}
      onClick={onClick}
      label={text}
      disabled={false}>
      <StyledIcon src={icon} disabled={false} />
      <StyledSmallViewTip>{text}</StyledSmallViewTip>
    </StyledToolbarItem>
  )
```